roles/pulp_database/tasks/main.yml

```diff
@@ -13,31 +13,8 @@
   tags:
     - always
 
-- include_tasks: install_postgres.yml
-  when:
-    - pulp_install_db |bool
-
-- meta: flush_handlers
-
-- block:
-
-    - name: Run database auth migrations
-      command: '{{ pulp_django_admin_path }} migrate auth --no-input'
-      register: migrate_auth
-      changed_when: "'No migrations to apply' not in migrate_auth.stdout"
+- name: Set database defaults if not provided
+  set_fact:
+    merged_pulp_settings: "{{ pulp_settings_db_defaults|combine(pulp_settings, recursive=True) }}"
```

Comment:
Is it duplicating https://github.com/pulp/pulp_installer/blob/master/roles/pulp_common/tasks/configure.yml#L3 ?

Reply:
Yes, but by necessity.

The pulp _database role no longer depends on the pulp_common role.

Reply:
gotcha! Thank you!